### PR TITLE
API-8177 pagination described

### PR DESCRIPTION
**Pull request by czeczot**
## 🚀 Links

- [[Feature branch]](https://developers.labs.livechat.com/docs/feature/API-8177-pagination)
- [[Jira]](https://livechatinc.atlassian.net/browse/API-8094)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/messaging/agent-chat-api/index.mdx [hunk 1]
```diff
@@ -479,6 +479,26 @@ curl -X POST \
...
 </Section>
 
+## Pagination
+
+<Section>
+
+<Text>
+
+Pagination is a mechanism that allows splitting the database output into more manageable chunks of data.
+Basing on `limit` and `sort order` parameters pagination is able to decide how many records will be returned at once and if it should fetch oldest or newest data first.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx:489]:
```suggestion
Based on the `limit` and `sort order` parameters, pagination is able to decide how many records will be returned at once and whether it should fetch the oldest or the latest data first.
```

File: content/messaging/agent-chat-api/index.mdx [hunk 2]
```diff
@@ -479,6 +479,26 @@ curl -X POST \
...
+
+Any filters that could be applied, should be provided in first pagination request. In response we can get `next_page_id` and `previous_page_id` parameters.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx:491]:
```suggestion
Any filters that could be applied should be provided in the very first pagination request. In the response, you'll get the `next_page_id` and `previous_page_id` parameters.
```

File: content/messaging/agent-chat-api/index.mdx [hunk 3]
```diff
@@ -479,6 +479,26 @@ curl -X POST \
...
+Next request should be performed using one of these parameters as `page_id` depending on iterations direction - forward or backward.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx:492]:
```suggestion
You should make the subsequent request using one of these parameters as `page_id`, depending on the direction of iteration: forward or backward.
```

File: content/messaging/agent-chat-api/index.mdx [hunk 4]
```diff
@@ -479,6 +479,26 @@ curl -X POST \
...
+
+`filters`, `limit` and `sort_order` parameters can not be provided along with `page_id`.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx:494]:
```suggestion
The `filters`, `limit`, and `sort_order` parameters can't be provided along with `page_id`.
```

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

File: content/messaging/agent-chat-api/v3.3/index.mdx [hunk 5]
```diff
@@ -50,6 +50,18 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
...
+
+Pagination is a mechanism that allows splitting the database output into more manageable chunks of data.
+Based on the `limit` and `sort order` parameters, pagination is able to decide how many records will be returned at once and whether it should fetch the oldest or the latest data first.
+
+Any filters that could be applied should be provided in first pagination request. In the response, you'll get the `next_page_id` and `previous_page_id` parameters.
+You should make the subsequent request using one of these parameters as `page_id`, depending on the direction of iteration: forward or backward.
+
+The `filters`, `limit`, and `sort_order` parameters can't be provided along with `page_id`.
+
+The maximum duration of the `page_id` parameter before it expires is one month.
```
Comment (Adimus11) [inline content/messaging/agent-chat-api/v3.3/index.mdx:63]: I think it would be nice to wrap this information in some `notice`

**Review by Adimus11 — APPROVED** [review]

File: content/messaging/agent-chat-api/v3.3/index.mdx (hunk 5 above)
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/v3.3/index.mdx:63]: Good idea. We could use the :bulb: emoji (docs support it)

**Pull request merged**